Pages/Healthcare.cshtml

```diff
@@ -1,24 +1,25 @@
 ﻿@page
 @model IS7024_Project.Pages.HealthcareModel
 @{
+    ViewData["title"] = "Healthcare Information";
     var healthcares = (QuickType1.Healthcare[])ViewData["Healthcare"];
 }
 
 <script>
-    function myFunction() {
+    function filterZipCode() {
         // Declare variables
         var input, filter, table, tr, td, i, txtValue;
         input = document.getElementById("myInput");
-        filter = input.value.toUpperCase();
+       filter = input.value.toUpperCase();
         table = document.getElementById("healthcaretable");
         tr = table.getElementsByTagName("tr");
 
         // Loop through all table rows, and hide those who don't match the search query
         for (i = 0; i < tr.length; i++) {
-            td = tr[i].getElementsByTagName("td")[5];
+            row_info = tr[i].getElementsByTagName("td")[5];
```

Comment:
changed variable name

Reply:
Go one step further.  What is in these rows?  Name the variable after that.